docs/sources/setup-grafana/installation/rpm/index.md

```diff
@@ -2,29 +2,23 @@
 aliases:
   - ../../installation/installation/rpm/
   - ../../installation/rpm/
-description: Grafana Installation guide for RPM-based Linux, such as Centos, Fedora,
-  OpenSuse, and Red Hat.
-title: Install on RPM-based Linux
+description: Install guide for Grafana on Red Hat, RHEL, and Fedora.
+title: Install Grafana on Red Hat, RHEL, or Fedora
+menuTitle: Redhat, RHEL, or Fedora
 weight: 400
 ---
 
-# Install on RPM-based Linux (CentOS, Fedora, OpenSuse, Red Hat)
+# Install Grafana on Red Hat, RHEL, or Fedora
 
-This topic explains how to install Grafana dependencies, download and install Grafana, get the service up and running on your RPM-based Linux system, and the installation package details.
+This topic explains how to install Grafana dependencies, install Grafana on Redhat, RHEL, or Fedora, and start the Grafana server on your system.
 
-## Repository migration (November 8th 2022)
+You can install Grafana using a YUM repository, using RPM, or by downloading a binary `.tar.gz` file.
 
-From that date, Grafana packages will be served from a new repository (<packages.grafana.com/rpm/{product}> -> <rpm.grafana.com>). The new repository serves, from a single YUM/DNF configuration, all Grafana OSS products, as well as Grafana Enterprise.
+If you install via the `.tar.gz` file, then you must manually update Grafana for each new version.
 
-The old URLs will still work, serving the content from the new repository, but you may encounter warnings about some repository attributes changing.
+## Install Grafana using YUM
 
-## Note on upgrading
-
-While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "../../../upgrade-guide/" >}}) for tips and guidance on updating an existing installation.
-
-## 1. Download and install
-
-You can install Grafana from a YUM repository, manually using YUM, manually using RPM, or by downloading a binary `.tar.gz` file.
+You can install Grafana from a YUM repository or manually using YUM.
```

Comment:
Let's split this up so that the repository option is standalone.  the manual yum install is more closely related to the manual rpm install than to using the repo, and has the same upgrade limitation as rpm or tar.gz install

Reply:
Done.